src/viewer.js

```diff
@@ -142,7 +165,11 @@ module.exports = class Viewer {
   render () {
 
     this.renderer.render( this.scene, this.activeCamera );
-
+    if (this.state.grid) {
+      this.axisCamera.position.copy(this.defaultCamera.position)
+      this.axisCamera.lookAt(this.axisScene.position)    
+      this.axisRenderer.render( this.axisScene, this.axisCamera );
```

Comment:
Could we keep the axis at constant scale, regardless of the distance of the camera from the origin? With large models I'm seeing this result:

![Screen Shot 2019-09-23 at 10 56 34 PM](https://user-images.githubusercontent.com/1848368/65485285-553b1580-de56-11e9-82a8-ff1d4a399ad4.png)

Reply:
Using the size of the object, to size the axis.  Should be constant scale now.